scrapy/http/request/__init__.py

```diff
@@ -25,6 +25,12 @@ def NO_CALLBACK(*args, **kwargs):
     :class:`~scrapy.http.Request`, it indicates that the request is not meant
     to have a spider callback at all.
 
+    For example:
+
+    .. code-block:: python
+
+       Request("https://example.com", callback=NO_CALLBACK)
```

Comment:
I thought it best to add an example to make it clear that `()` should not be appended to the function name when assigning it to the `callback` parameter.